src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForGetOngoingTasks.cs

```diff
@@ -27,23 +27,33 @@ protected override Task HandleRemoteNodeAsync(ProxyCommand<OngoingTasksResult> c
 
         protected override RavenCommand<OngoingTasksResult> CreateCommandForNode(string nodeTag)
         {
-            return new GetOngoingTasksInfoCommand();
+            return new GetOngoingTasksInfoCommand(nodeTag);
         }
 
         protected override bool SupportsCurrentNode => false;
     }
 
     internal class GetOngoingTasksInfoCommand : RavenCommand<OngoingTasksResult>
     {
+        private readonly string _nodeTag;
+
+        public GetOngoingTasksInfoCommand(string nodeTag = null)
+        {
```

Comment:
Should we use `SelectedNodeTag` here instead?

Reply:
Done